docs/flowcharts/flowchart_utils.md

```diff
@@ -0,0 +1,24 @@
+# content/shared
+
+This flowchart represents the high-level interaction of components under the `utils`, `release_tools`, `ssg` and `Dockerfiles` folders.
+
+Note that colors are used to highlight components which interact with other folders under the `content` repository.
+
+> ***NOTE***: Since the colors have only visual effects when presented in individual flowcharts, they are very useful when analyzing multiple flowcharts together.
+
+<div class="mermaid" style="width=100%;">
+flowchart TD
+    subgraph utils
+    200[utils] --> |contains| 201(scrits)
+    200[utils] --> |contains| 202(html templates)
+    203[release_tools] --> |contains| 204(scripts)
+    205[ssg] --> |contains| 206(python scripts)
```

Comment:
The `ssg` directory is a Python module providing a lot of classes and functions, it isn't a script that could be executed.